babeld/babel_interface.c

```diff
@@ -484,7 +484,9 @@ DEFUN (babel_set_rtt_min,
     babel_ifp = babel_get_if_nfo(ifp);
     assert (babel_ifp != NULL);
 
-    babel_ifp->rtt_min = rtt;
+    babel_ifp->rtt_min =
+	    rtt
+	    * 1000; // value entered in milliseconds and stored as microseconds
```

Comment:
Can we prefer using C-style comments `/* */`?

Reply:
Sure. Done.